src/main/java/frc/team4909/robot/sensors/PhotoElectricSensors.java

```diff
@@ -0,0 +1,75 @@
+package frc.team4909.robot.sensors;
+import frc.team4909.robot.Robot;
+import edu.wpi.first.wpilibj.DigitalInput;
+public class PhotoElectricSensors{  //not sure if extends or implements
+    public static DigitalInput frontLeftSensor,     
+                              frontMiddleLeftSensor, 
+                              frontMiddleRightSensor,
+                              frontRightSensor;
+    private double velocity = 0.3;
+    boolean state = false;
+    boolean state2 = false;
+    public PhotoElectricSensors(){
+        super();
+        frontLeftSensor = new DigitalInput(0);
+        BackLeftSensor = new DigitalInput(1);
+        frontRightSensor = new DigitalInput(2);
+        BackRightSensor = new DigitalInput(3);
+    }
+
+    public void onLine(){
+        
+    if(!frontLeftSensor.get() && !frontRightSensor.get()){
+        state=true;
+    }
+    else {
+        Robot.myDrive.tankDrive(velocity, velocity);
+        lineFollow();
+    }
+    //  if(!frontMiddleLeftSensor.get()){
+    //    System.out.println("Middle Left Sensor is ON LINE");
+    //    lineFollow();
+    //  }  
+    //  if(!frontMiddleRightSensor.get()){
+    //      System.out.println("Middle Right Sensor is ON LINE");
+    //      lineFollow();
+    //     }
+     
+    }
+    
+    
+    //Case 1: Coming in from right side at an angle
+    public void lineFollow(){
+      boolean backleft = BackLeftSensor.get();
+      boolean backright = BackRightSensor.get();
+      boolean frontleft = frontLeftSensor.get();
+      boolean frontright = frontRightSensor.get();
```

Comment:
Do we have four sensors on the robot? Or really, do we have any intention of putting four sensors on the robot?